roles/openshift_metrics/tasks/import_jks_certs.yaml

```diff
@@ -87,32 +34,23 @@
   register: jgroups_keystore
   check_mode: no
 
-- name: create JKS pod
+- name: create JKS container
   command: >
-    {{ openshift.common.client_binary }}
-    --config={{ mktemp.stdout }}/admin.kubeconfig
-    -n {{openshift_metrics_project}}
-    create -f {{mktemp.stdout}}/jks_pod.yaml
-    -o name
-  register: podoutput
-  check_mode: no
-  when: not metrics_keystore.stat.exists or
-        not metrics_truststore.stat.exists or
-        not cassandra_keystore.stat.exists or
-        not cassandra_truststore.stat.exists or
-        not jgroups_keystore.stat.exists
-
-- command: >
```

Comment:
I we can't use `oc` here, but shouldn't we wait until the docker container terminates still?